src/pages/ManageWallet/ManageWallet.tsx

```diff
@@ -40,20 +42,24 @@ import useInterval from 'hooks/useInterval';
 import { REQUEST_REFRESH_INTERVAL_MS } from 'constants/request';
 import { SupportedNetworkName } from 'constants/network';
 import { PageLogo, PageHeadline } from 'styles/common';
+import TransactionBox from './TransactionBox';
 
 interface StateProps {
 	gasPrice: GasPrice;
 	walletInfo: DelegateWalletInfo;
 	isLoading: boolean;
 	walletAddr: WalletAddress;
-	transactions: Transaction[];
+	mintTransaction?: Transaction;
+	burnTransaction?: Transaction;
+	claimTransaction?: Transaction;
 	errors: TransactionError[];
 	networkName: SupportedNetworkName;
 }
 
 interface DispatchProps {
 	fetchDelegateWalletInfoRequest: typeof fetchDelegateWalletInfoRequest;
 	addTransaction: typeof addTransaction;
+	removeTransaction: typeof removeTransaction;
 	addError: typeof addError;
 	removeError: typeof removeError;
 }
```

Comment:
guess this would all be removed now with `ConnectedProps`! (no need to do this refactor now)